_docs/setup/kubernetes/multicluster-install.md

```diff
@@ -138,7 +153,7 @@ install one:
 1. Install the Helm chart:
 
    ```bash
-   helm install install/kubernetes/helm/istio-remote --name istio-remote --set pilotEndpoint=`pod_ip_of_pilot_local` --set policyEndpoint=`pod_ip_of_policy_local` --set statsdEndpoint=`pod_ip_of_statsd_local` > $HOME/istio-remote.yaml -n istio-system
+   helm install install/kubernetes/helm/istio-remote --name istio-remote --set global.pilotEndpoint=${PILOT_POD_IP} --set global.policyEndpoint=${POLICY_POD_IP} --set global.statsdEndpoint=${STATSD_POD_IP} --namespace istio-system
    ```
 
 ### Helm configuration parameters
```

Comment:
Are the config params needed for both alternative or just the last one? What is the user supposed to do with these config, define them or what? Shouldn't this be before any helm command is executed?